config/src/config/mod.rs

```diff
@@ -81,8 +81,6 @@ pub struct NodeConfig {
     #[serde(default)]
     pub test: Option<TestConfig>,
     #[serde(default)]
-    pub upstream: UpstreamConfig,
```

Comment:
The k8s configs will need to be updated to remove this config.

Reply:
I suggested to Josh, that he separate removing this config from the changes.  Given the extra complexity with going through other repos and configs to remove it

Reply:
Thanks, all! Yes, I'm going to separate that out into a follow up PR after we land this. Just so that this one isn't blocked.